perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -7838,6 +7842,10 @@ protected void initialize() {
 		//Prepare all attribute definition from system perun
 		Set<AttributeDefinition> allAttributesDef = new HashSet<>(this.getAttributesDefinition(sess));
```

Comment:
Using ``allAttributesDef`` probably doesn't ensure, that all definitions are created, since some attributes are created by registrar and cabinet components and they are probably initialized later, since they depend on perun bean. Please move attribute definitions creation from registrar and cabinet into this init method.

Reply:
Since the method `createAttribute()` was changed and now it also tries to initialize the module. And the cabinet/registrar attributes are created through this method, I think it should be fine.

Reply:
Although it may be faster if they were moved into this init method... Should I move them?

Reply:
No, you are right, I didn't realize that. It should be fine.